### PR TITLE
[debug dump] Refactoring Modules and Unit Tests

### PR DESCRIPTION
**Pull request by vivekreddynv**
#### What I did

1) Moved the "populate_mock" method to dump/helper.py i.e. common to all the tests
2) Moved the "add_to_ret_template" to Executor class i.e. common across all the modules.

#### How to verify it

```
Unit Tests's:

vkarri@96050efd4f16:/sonic/src/sonic-utilities$ pytest-3 tests/dump_tests/ 
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.9.2, pytest-6.0.2, py-1.10.0, pluggy-0.13.0
rootdir: /sonic/src/sonic-utilities/tests, configfile: pytest.ini
plugins: pyfakefs-4.5.3, cov-2.10.1
collected 116 items                                                                                                                                                        

tests/dump_tests/dump_state_test.py .............                                                                                                                    [ 11%]
tests/dump_tests/match_engine_test.py ...............................                                                                                                [ 37%]
tests/dump_tests/module_tests/copp_test.py ..........                                                                                                                [ 46%]
tests/dump_tests/module_tests/evpn_test.py ......                                                                                                                    [ 51%]
tests/dump_tests/module_tests/port_test.py ......                                                                                                                    [ 56%]
tests/dump_tests/module_tests/portchannel_member_test.py ...                                                                                                         [ 59%]
tests/dump_tests/module_tests/portchannel_test.py ....                                                                                                               [ 62%]
tests/dump_tests/module_tests/route_test.py ..........                                                                                                               [ 71%]
tests/dump_tests/module_tests/vlan_test.py .....................                                                                                                     [ 89%]
tests/dump_tests/module_tests/vxlan_tunnel_map_test.py .....                                                                                                         [ 93%]
tests/dump_tests/module_tests/vxlan_tunnel_test.py .......                                                                                                           [100%]

=========================================================================== 116 passed in 1.21s ============================================================================
```

### CODE REVIEW

**Comment by vivekreddynv** [comment]
@SuvarnaMeenakshi, Can you help review

**Comment by dgsudharsan** [comment]
@SuvarnaMeenakshi Your code review comments are handled here. Can you please review and signoff?

File: dump/plugins/executor.py [hunk 1]
```diff
@@ -24,3 +25,14 @@ def execute(self, params):
...
     def get_all_args(self, ns):
         pass
+    
+    def add_to_ret_template(self, table, db, keys, err, add_to_tables_not_found=True):
+        if db not in self.ret_temp:
+            return []
+
+        if not err and keys:
+            self.ret_temp[db]["keys"].extend(keys)
+            return keys
```
Comment (SuvarnaMeenakshi) [inline dump/plugins/executor.py:35]: The return value is different from what was there in the older add_to_ret_template.
Is there any reason for the change? This return value is not being used anywhere.

Comment (vivekreddynv) [reply dump/plugins/executor.py:35]: Previously, different modules has different implementations of the same function. Route module was using the keys returned by this function to do further processing. The other modules were just using this as a boolean return.

 An empty list is Falsy, so this approach is more general

**Review by SuvarnaMeenakshi — APPROVED** [review]
lgtm

**Pull request merged**